src/Randomizer.SMZ3/ILogic.cs

```diff
@@ -55,5 +55,11 @@ public interface ILogic
         public bool CanPassFireRodDarkRooms(Progression items);
 
         public bool CanParlorSpeedBoost(Progression items);
+
+        public bool CanPassMockBallRoom(Progression items);
```

Comment:
Nitpicking, but I feel like the logic shouldn't be named after the trick if it can return true when you have the trick disabled.

One solution would be to rename this to `CanPerformMockball` and adding `|| items.SpeedBooster` to everything, but I think I'd prefer this to be called something sillier like `CanMoveAtHighSpeeds` or something more elegant.

Reply:
Hmm... I must have misunderstood your previous suggestion, I suppose. I'll make these updates and update to 4.0.